LucidGloves/src/ControllerDriver.cpp

```diff
@@ -89,40 +94,48 @@ void ControllerDriver::StartDevice() {
 	//DebugDriverLog("Getting ready to connect:");
 	if (m_communicationManager->IsConnected()) {
 		//DebugDriverLog("Connected successfully");
-		m_communicationManager->BeginListener([&](VRCommData_t datas) {
-
+		m_communicationManager->BeginListener([&](std::string receivedString) {
```

Comment:
I'm thinking we could perhaps pass the encoding interface to the communication manager and for it to return call it and return `VRCommData_t` like before?